packages/server/src/index.ts

```diff
@@ -60,17 +61,16 @@ export const start = async (
 
   app.use(Express.static(outDir));
 
-  fs.watch(process.cwd(), {recursive: true}, async (_, filename) => {
-    // TODO: Try/catch?
```

Comment:
Does chokidar handle async functions well?

Reply:
it doesn't do anything with return value so I don't think it matters. It just expects a function with signature `(string, fs.Stats) => void`

Reply:
In that case, please handle potential exceptions instead of just deleting the TODO.